### ISSUE TITLE
강의 댓글 질문에 대한 이해

### ISSUE THREAD

**Issue by Mirlam8** [liganega/CodingRG#26]
6-2 lec 강의 에서 
(https://www.youtube.com/watch?v=jMU9G5WEtBc&feature=youtu.be)
댓글로 추가된 질문과 답변에 대해 이해가 잘 되지 않습니다.
-Q- Junghoon Kim
L = [0, 1]
Y1 = [0, 1]
Y2 = [1, 0]
Cross-entropy cost function 식 자체가 L을 포함하고 있기 때문에
L 이 0 일 경우에 Y가 어떠한 값이 나오더라도 cost 에 더해지지가 않는 것 같아서요. 
이게, 항상 one-hot encoding 기법을 사용할 경우에만 사용되어지는 것인가요?
즉, Y3 = [0.9, 1] 혹은 Y4= [0.1, 1] 일 경우에 항상 cost 는 0가 되어지는데 이러한 경우에 문제가 없는것인가요?
학습을 시키는 메커니즘에서 타겟을 1으로 레이블 되어있는 영역에만 집중을 하는것으로도 생각이 되어지는데, 타겟이 0이어도 hypothesis 가 0에 근접하도록 학습도 되어야 하는 것 아닌가요? 
-A- Sung Kim
매우 잘 이해하신듯 합니다. softmax 에서는 "항상 one-hot encoding "을 사용해야 합니다. 만약 다르게 하시면 cost도 새롭게 설계하셔야 합니다.﻿

**Comment by Mirlam8** [comment]
Hypothesis 에 따라 맞는 cost 설계를 사용해야 합니다.
